Code-review.md

```diff
@@ -0,0 +1,64 @@
+Code review is an important part of building quality software.  We review each
+other's code in order to:
+
+* Keep tabs on what's changing to build and maintain shared understanding of
+  our systems.
+
+* Solicit and incorporate the knowledge and experience of the team to improve
+  the code and understanding of the problem being solved.
+
+* Reduce the number of bugs or design flaws that make it into production to
+  avoid outsized impacts from only uncovering them later.
+
+Shared understanding is the most important part of code review.  It is nice to
+avoid bugs and flaws, but as it is impossible to avoid them entirely, it's
+important to focus on work that makes fixing problems _easier_.
+
+
+## Guidelines
+
+Code review interactions should primarily foster discussion and aim to produce
+rough consensus or agreement.  Suggesting changes is ok, but code reviews are
+much more than creating punch list of changes required by a gatekeeper.
+
+Communication is hard, especially via text-only mediums, and constructive
+critique doubly so.  Be kind and generous and give people the benefit of the
+doubt.  Try to over communicate rather than under communicate.
+
+Try to be timely, but realize that sometimes it's worth taking time to get
+feedback and do it right.  Talk explicitly about that tradeoff, as it is
+different case to case.  It's ok to ping people if PRs aren't getting
+attention.
+
+### As an author
+
+* Direct reviewers' attention to any parts you're unsure of or specific topics
+  on which you'd like feedback.
+
+* Follow up with reviewer comments to acknowledge them.  This can be as quick
+  as a reaction emoji or as open-ended as a discussion about the comment.
+
+* Wait for at least one approval from another team member before merge and
+  deploy.  More reviews are always ok!  Use your judgment if you think input
+  from more team members would be worthwhile for any reason (for example, the
+  scope, scale, intricacy of the changes).
+
+* Close the loop with the reviewer after implementing suggested changes.  Check
+  back with the person who suggested the changes and see if your solution
+  matches what they expected.  This checks for shared understanding and
+  miscommunication.
+
+### As a reviewer
+
+* Be explicit about which of your suggested changes you consider merge blockers
```

Comment:
Is there an easy way we can do this? 
In the "Review changes" workflow in github the option is listed as
"Request changes
Submit feedback that must be addressed before merging."
When I see requests in there I read them as required before merging.

For suggestions that are not merge blockers would it make sense to add the comments into the comments box at the center bottom of the page when you're on the Conversation view?

Reply:
GitHub offers three statuses for the overall state of the review (i.e. not tied to specific review comments):

- Commenting only
- Requesting changes
- Approving as-is

A review may have comments that are both blockers and not blockers, but the overall review status can't indicate which those are.

My tendency would be to leave this to the flexibility of written words instead of trying to codify it into a process.